webpack.mix.js

```diff
@@ -41,22 +41,33 @@ mix.js(src`scripts/app.js`, 'scripts')
 mix.copyDirectory(src`images`, publicPath`images`)
    .copyDirectory(src`fonts`, publicPath`fonts`);
 
-// Autoload
-mix.autoload({
-  jquery: ['$', 'window.jQuery'],
-});
-
 // Options
 mix.options({
   processCssUrls: false,
 });
 
-// Source maps when not in production.
-if (!mix.inProduction()) {
-  mix.sourceMaps();
-}
+// Generate source maps when not in production
+mix.sourceMaps(false);
+
+// Hash and version files in production
+mix.version();
 
-// Hash and version files in production.
-if (mix.inProduction()) {
-  mix.version();
-}
+// Remove unused styles in production
+mix.purgeCss({
+  content: [
+    './resources/views/**/*.php',
+    './resources/assets/**/*.js',
+  ],
+  defaultExtractor: content => content.match(/[A-Za-z0-9-_:/]+/g) || [],
+
+  /*
+   | Add classes, IDs, or selectors that which do not exist in your views or
+   | scripts. By default, We have whitelisted classes generated by WordPress.
+   */
```

Comment:
Again, maybe TMI, but I wanted to add some points for how the developer could use the rules since they might be baffling if left without mention.